src/traces/candlestick/attributes.js

```diff
@@ -0,0 +1,49 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Lib = require('../../lib');
+var OHLCattrs = require('../ohlc/attributes');
+var boxAttrs = require('../box/attributes');
+
+var directionAttrs = {
+    visible: {
+        valType: 'enumerated',
+        values: [true, false, 'legendonly'],
+        role: 'info',
+        dflt: true,
+        description: [
+
+        ].join(' ')
+    },
+
+    color: Lib.extendFlat({}, boxAttrs.line.color),
+    width: Lib.extendFlat({}, boxAttrs.line.width),
+    fillcolor: Lib.extendFlat({}, boxAttrs.fillcolor),
+    tickwidth: Lib.extendFlat({}, boxAttrs.whiskerwidth, { dflt: 0 }),
```

Comment:
EDIT  following a quick chat with @alexcjohnson :

Let's go with:

- `tickwidth` for OHLC as per the wiki [article](https://en.wikipedia.org/wiki/Open-high-low-close_chart)
- `whiskerwidth` for candlestick on par with our `box` attributes
`whiskerwidth` for